crates/wiggle/crates/generate/src/types/handle.rs

```diff
@@ -17,6 +17,12 @@ pub(super) fn define_handle(
         #[derive(Copy, Clone, Debug, ::std::hash::Hash, Eq, PartialEq)]
         pub struct #ident(u32);
 
+        impl #ident {
+            pub unsafe fn inner(&self) -> u32 {
+                self.0
+            }
+        }
+
```

Comment:
can you just use the `Into<u32>` implementation?

Reply:
I agree it'd be good to have one way to do this, rather than two. Could I suggest preferring `.inner()` over `.into()`? Most of the code in wasi-common should treat these as opaque, so it'd be nice for the places where we do need to peek inside to stand out.

Reply:
Oh, this must be a remnant of my hack to not stifle progress with `wasi-common` while some required changes to `wiggle` were in progress. This change is no longer part of this PR, so please ignore it.